docs/src/main/paradox/dispatchers.md

```diff
@@ -44,6 +44,17 @@ You can read more about parallelism in the JDK's [ForkJoinPool documentation](ht
 
 When Running on Java 9+, you can use `maximum-pool-size` to set the upper bound on the total number of threads allocated by the ForkJoinPool.
 
+**Experimental**: When Running on Java 21+, you can use `virtualize=on` to enable the virtual threads feature.
+When using virtual threads, all virtual threads will use the same `unparker`, so you may want to 
+increase the number of `jdk.unparker.maxPoolSize`. 
+
+NOTE: THIS IS AN ADVANCED OPTION, USE WITH CAUTION.
+REQUIREMENTS:
```

Comment:
Can you use a markdown header for requirements? No need to shout case it either.

I would also remove the shout casing on the Note above - maybe the word NOTE can shouted but it could just be 'NB' (common abbreviation for https://en.wiktionary.org/wiki/nota_bene#English).